version.go

```diff
@@ -0,0 +1,4 @@
+package main
+
+// Version is the current version of Glide
+const Version = "0.13.0-dev"
```

Comment:
Might be useful for exterior libs to be able to get at this, in the case that they use glide as a lib.